### DISCUSSION TITLE
Where is libs.versions.toml

### DISCUSSION THREAD

**Discussion by hongshuai1994** [open-telemetry/opentelemetry-java#6545] [Q&A] [answered]
I want to compile and package this project locally, but I am not very familiar with gradle. I am trying to learn gradle and build it, but I did not find the [libs.versions.toml](https://docs.gradle.org/current/userguide/part3_gradle_dep_man.html#step_1_understanding_the_version_catalog) mentioned in the gradle document. Where is this file?

**Comment by hongshuai1994** [comment]
branch: release/v1.26.x

**Comment by jkwatson** [comment, marked as answer]
I'm pretty sure we never use anything like that. The gradle dependency versions are in the `dependencyManagement` module. 

I recommend reading the CONTRIBUTING.md in the root of the project to get started.
